pkg/virt-handler/vm.go

```diff
@@ -1272,9 +1272,13 @@ func (d *VirtualMachineController) checkNetworkInterfacesForMigration(vmi *v1.Vi
 		networks[network.Name] = network.DeepCopy()
 	}
 	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
-		if iface.Bridge != nil && networks[iface.Name].Pod != nil {
-			return fmt.Errorf("cannot migrate VMI with a bridge interface connected to a pod network")
+		if networks[iface.Name].Pod != nil {
```

Comment:
@vatsalparekh we check if a VMI is migrateable when it gets processed in virt-controller. If it is not migrateable, we add a `VirtualMachineInstanceIsMigratable` condition which is set to false. If that is present creating  migration objects for thes VMI will be rejected.

You will have to move this check to virt-controller.

Reply:
sorry it happens in virt-handler, but anyway, instead of blocking the migration at this place, we have to ensure that it never gets the condition that it is migrateable assigned by virt-handler.

Reply:
Ok, forget me previous comments. I was completely on the wrong track. Still @phoracek is bridge not also used to get a network connection to an external network with its own DHCP? Wouldn't we then not also block migrations for these?

Reply:
Ok, so the code does something different than intended.